var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -67,7 +67,7 @@ class Openspeedshop(CMakePackage):
 
     # Use when the git repository is available
     version('2.3.1.4', branch='2.3.1.4',
-            git='https://github.com/OpenSpeedShop/openspeedshop.git')
+            git='https://github.com/OpenSpeedShop/openspeedshop.git', preferred=True)
```

Comment:
This should already be the default version, no need for this change.

Reply:
Same as above:
I think it was picking 2.3.1.4 as you say without the preferred, but spack was not following the same path as if the preferred statement clause is present.  The build was getting concretization errors without the preferred clause.

Reply:
@adamjstewart I will retest this and the cbtf-argonavis-gui one too.